design-at-cornell/src/clubs/dashboard/ClubBubble.tsx

```diff
@@ -61,40 +65,42 @@ const ClubBubble = (club: Club) => {
     >
       <ModalContainer>
         <VerticalModalHeader>
-          <LargeTitle >{club.title}</LargeTitle>
+          <LargeTitle>{club.title}</LargeTitle>
           <Subtitle>
             <p style={{ color: 'white' }}>{'Apply by '}</p>
           </Subtitle>
           <TagsContainer>
-            {club.content.orgType === '' ? null : (
-              <Tag>{club.content.orgType}</Tag>
-            )}
-            {club.content.size === '' ? null : (
-              <Tag>{club.content.size}</Tag>
-            )}
+            {club.content.orgType === '' ? null : <Tag>{club.content.orgType}</Tag>}
+            {club.content.size === '' ? null : <Tag>{club.content.size}</Tag>}
             {club.content.designAreas.filter((x) => x !== '').length === 0 ? null : (
               <Tag>{club.content.designAreas.join(', ')}</Tag>
             )}
-            {club.content.credits === null ? null : (
-              <Tag>{club.content.credits + ' Credits'}</Tag>
-            )}
+            {club.content.credits === null ? null : <Tag>{club.content.credits + ' Credits'}</Tag>}
           </TagsContainer>
           <ButtonContainer>
             <LargeRectangularButton>
-              <img onClick={() => (window.location.href = club.content.website)} src={openlink} alt={'website'} />
-              <p>Website</p> 
+              <img
+                onClick={() => (window.location.href = club.content.website)}
```

Comment:
The Club Bubble looks great! Something to add on regarding the button is that when using the OnClick, it might be more beneficial of using window.open instead of window.location.href that way the user isn't redirected from the website.